modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+.Procedure
+
+. Create the `aws-load-balancer-operator` namespace:
+
+[source,terminal]
+----
+oc new-project aws-load-balancer-operator
+----
++
+The following configurations will be automatically created:
+
+.Expected `OperatorGroup` object
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+name: aws-load-balancer-operator
+namespace: aws-load-balancer-operator
+spec:
+upgradeStrategy: Default
+----
+
+.Expected `Subscription` object
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.1
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+. Apply the configurations by running the following command:
++
+[source,terminal]
+Suggested change
+----
+$ oc apply -f 
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+name: aws-load-balancer-operator
+namespace: aws-load-balancer-operator
+spec:
+upgradeStrategy: Default DO WE NEED THE FILENAME HERE
```

Comment:
This can be replaced by an "in-place" command without any additional file:
```
cat <<EOF | oc apply -f -
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  targetNamespaces: []
EOF
```

Reply:
Per our standards, we do one step at a time. :)

Reply:
Then you can split it in 2 commands:
```bash
cat <<EOF > filename.yaml
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  targetNamespaces: []
EOF
```

```bash
$ oc apply -f filename.yaml
```

Reply:
@alebedev87 Per our style guidelines, we avoid `cat` commands. Is there a way to rework this to only use `oc` commands? Thanks!

Reply:
Not really. I think we have to show the yaml and ask to save it into a file:

```yaml
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: aws-load-balancer-operator
  namespace: aws-load-balancer-operator
spec:
  targetNamespaces: []
```

The `oc apply -f <filename>` remains the same.